nipype/algorithms/confounds.py

```diff
@@ -945,3 +1022,69 @@ def _compute_tSTD(M, x, axis=0):
     stdM[stdM == 0] = x
     stdM[np.isnan(stdM)] = x
     return stdM
+
+
+# _cosine_drift and _full_rank copied from nipy/modalities/fmri/design_matrix
```

Comment:
Is there a a big disadvantage of depending on nipy here instead of copying the code?